.github/workflows/release.yaml

```diff
@@ -1,19 +1,43 @@
 name: Release
 
 on:
-  # Trigger this workflow when a tag is pushed in the format `v1.2.3`.
-  push:
-    tags:
-      # Pattern syntax: <https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet>.
-      - "v[0-9]+.[0-9]+.[0-9]+*"
-  # Trigger this workflow manually via workflow dispatch.
+  # Trigger this workflow on a manual workflow dispatch.
   workflow_dispatch:
     inputs:
       version:
         description: "Version number in the format `v1.2.3`"
         required: true
         type: string
-
+      build_for_windows:
+        description: "Build for Windows? (WARNING: very slow!)"
+        default: false
+        type: boolean
+      build_for_macos:
+        description: "Build for macOS?"
+        default: false
+        type: boolean
+      build_for_linux:
+        description: "Build for Linux?"
+        default: false
+        type: boolean
+      build_for_web:
+        description: "Build for web?"
+        default: false
+        type: boolean
+      upload_to_github:
+        description: "Upload to GitHub releases?"
+        default: true
+        type: boolean
+      upload_to_itch:
+        description: "Upload to itch.io?"
+        default: true
+        type: boolean
+      deny_warnings:
+        description: "Deny warnings?"
+        default: false
+        type: boolean
```

Comment:
I personally find it a bit strange that this is an option, as it doesn't seem to "fit" with the other options. I'm not sure what to do about it, though, and it's probably fine staying